types/npmlog/index.d.ts

```diff
@@ -2,65 +2,81 @@
 // Project: https://github.com/npm/npmlog#readme
 // Definitions by: Daniel Schmidt <https://github.com/DanielMSchmidt>
 //                 Zhu Zijia <https://github.com/littlepiggy03>
+//                 Joseph Wynn <https://github.com/wildlyinaccurate>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.4
+// TypeScript Version: 3.4
```

Comment:
Probably this is the minimal TypeScript version required for compiling this type definition, so don't need to edit it? (I'm not so sure.)

(See https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-use-features-from-typescript-21-or-above and https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-use-features-from-typescript-31-or-above)